web/test/factories.ts

```diff
@@ -500,11 +500,19 @@ export const randomNegativeFilteredIndustry = (func: (industry: Industry) => boo
   });
 };
 
-export const randomHomeBasedIndustry = (excludeIndustryId?: string): string => {
+export const randomHomeBasedIndustry = (
+  excludeIndustryId?: string,
+  excludeSecondIndustryId?: string
+): string => {
```

Comment:
[dust] I'd like the props for this to be more explicit than positional args. 

What if we turn the prop for `randomHomeBasedIndustry()` into an object with a key `industriesToExclude` whos value is an array of industry ids?

```typeScript
randomHomeBasedIndustry({
  industriesToExclude: ["acupuncture"]
})
```